Post/What is WAI? The Web Accessibility Initiative

ilknureren opened the pull request:
Post Idea: https://github.com/a11yproject/a11yproject.com/issues/1449

### Your idea
I will be writing a post on WAI, Web Accessibility Initiative. I will focus on its history and importance.

### Outline (optional)
What is the World Wide Consortium?
What is the Web Accessibility Initiative?
WCAG and WAI-Aria Recommendations:
Working groups/interest groups
Conclusion

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+W3C launched The Web Accessibility Initiative, (WAI), in 1997, with support from the White House. The Web Accessibility Initiative’s main goal is to make the internet accessible for people with disabilities.
+
+WAI brings people from disability organizations, governments and research labs from all around the world to develop guidelines and resources to make the Web accessible to people with disabilities that include auditory, cognitive, neurological, physical, speech and visual disabilities.
```
```suggestion
WAI brings people from disability organizations, governments, and research labs from all around the world to develop guidelines and resources to make the Web accessible to people with disabilities. This includes: auditory, cognitive, neurological, physical, speech and visual disabilities.
```

This sentence was a little long, so I'm suggesting we split it into two smaller sentences.

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+As described by [W3C Website](https://www.w3.org/WAI/about/), WAI “pursues accessibility of the web through these primary activities”:
+
+- ensuring that W3C standards support accessibility
+- developing accessibility guidelines for web content and applications,
+  browsers, and authoring tools
+- developing resources to improve web accessibility evaluation
+  processes and tools
+- supporting education and outreach on web accessibility
+- coordinating with research and development that may impact future  
+  accessibility of the web
+- promoting harmonized international uptake of web accessibility  
+  standards
```
```suggestion
- Ensuring that W3C standards support accessibility
- Developing accessibility guidelines for web content and applications,
  browsers, and authoring tools
- Developing resources to improve web accessibility evaluation
  processes and tools
- Supporting education and outreach on web accessibility
- Coordinating with research and development that may impact future  
  accessibility of the web
- Promoting harmonized international uptake of web accessibility  
  standards
```

We capitalize our bullet points, so updating that here.

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+[Web Content Accessibility Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/), or WCAG is a benchmark for web accessibility. WCAG was created with a goal to clearly define technical standards and unifying accessibility standards. WCAG’s standards are categorized under 4 principles.
+
+- Perceivable:
```
```suggestion
### Perceivable
```

Changing these to `h3` headings, as an isolated bullet point could be confusing.

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+  Users should be able to identify content and elements by means of senses. For example, if a website's navigation changes from page to page, users will have to relearn the navigation of the site, which will not be a great user experience.
+
+- Operarable:
```
```suggestion
### Operarable:
```

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+  Users should successfully use controls, buttons, navigation, and other interactive elements on the page. One example of this would be that websites should be operable by a keyboard and the page should have the same functionalities it has when used by a mouse to navigate.
+
+- Understandable
```
```suggestion
### Understandable
```

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+  Users should be able to comprehend the content and learn from your website. One example of this would be if abbreviations or jargons are used, they should be defined for the user.
+
+- Robust:
```
```suggestion
### Robust:
```

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+- Perceivable:
+
+  Users should be able to identify content and elements by means of senses. For example, if a website's navigation changes from page to page, users will have to relearn the navigation of the site, which will not be a great user experience.
```
```suggestion
Users should be able to identify content and elements by means of senses. For example, if a website's navigation changes from page to page, users will have to relearn the navigation of the site, which will not be a great user experience.
```

Also removing the indents.

ericwbailey replied:
The W3C may use the term "user" or "users", but [we use the term "person" or "people"](https://www.a11yproject.com/content-style-guide/#user-or-person-). Can we rephrase instances where "user" and "users" is used to use our preferred terminology instead?

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+- Operarable:
+
+  Users should successfully use controls, buttons, navigation, and other interactive elements on the page. One example of this would be that websites should be operable by a keyboard and the page should have the same functionalities it has when used by a mouse to navigate.
```
```suggestion
Users should successfully use controls, buttons, navigation, and other interactive elements on the page. One example of this would be that websites should be operable by a keyboard and the page should have the same functionalities it has when used by a mouse to navigate.
```

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+- Understandable
+
+  Users should be able to comprehend the content and learn from your website. One example of this would be if abbreviations or jargons are used, they should be defined for the user.
```
```suggestion
Users should be able to comprehend the content and learn from your website. One example of this would be if abbreviations or jargon is used, they should be defined for the user.
```

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+- Robust:
+
+  Websites should be able to be accessed by a wide variety of users and technologies. For example, websites should be accessible for screen readers and many different browser versions.
```
```suggestion
Websites should be able to be accessed by a wide variety of users and technologies. For example, websites should be accessible for screen readers and many different browser versions.
```

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+- promoting harmonized international uptake of web accessibility  
+  standards
+
+Two best examples of WAI’s significance in everyday technology life are the Web Content Accessibility Guidelines, (WCAG) and Accessible Rich Internet Applications (WAI-Aria) Recommendations that were created by members of WAI.
```
```suggestion
Two best examples of WAI’s significance in everyday technology life are the Web Content Accessibility Guidelines, (<abbr>WCAG</abbr>) and Accessible Rich Internet Applications (<abbr>WAI-ARIA</abbr>) Recommendations that were created by members of WAI.
```

Noting the first use of these abbreviations here, as well.

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+Two best examples of WAI’s significance in everyday technology life are the Web Content Accessibility Guidelines, (WCAG) and Accessible Rich Internet Applications (WAI-Aria) Recommendations that were created by members of WAI.
+
+## Web Content Accessibility Guidelines, (WCAG) and Accessible Rich Internet Applications (WAI-Aria) Recommendations:
```
```suggestion
## Web Content Accessibility Guidelines, (WCAG) and Accessible Rich Internet Applications (WAI-ARIA) Recommendations:
```

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+  Websites should be able to be accessed by a wide variety of users and technologies. For example, websites should be accessible for screen readers and many different browser versions.
+
+Each WCAG guideline has success criteria broken down to three standards, A, AA, AAA, with triple A being the standard we should aim for.
```
```suggestion
Each WCAG guideline has success criteria broken down to three standards, A, AA, and AAA. AA is considered ideal support, and AAA is typically reserved for specialized audiences.
```

Updating this to clarify AA and AAA support. It's nice to strive for AAA support, but can actually be harmful if done not having close working knowledge of the access needs of the audience you serve.

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+  Websites should be able to be accessed by a wide variety of users and technologies. For example, websites should be accessible for screen readers and many different browser versions.
+
+Each WCAG guideline has success criteria broken down to three standards, A, AA, AAA, with triple A being the standard we should aim for.
+
```
It feels like we should have a heading here, as we transition from talking about the WCAG to ARIA.

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+Each WCAG guideline has success criteria broken down to three standards, A, AA, AAA, with triple A being the standard we should aim for.
+
+[Accessible Rich Internet Applications](https://www.w3.org/WAI/standards-guidelines/aria/), or WAI-ARIA is also a guideline that was created by WAI to increase accessibility. In particular, WAI-ARIA gives guidance on how to create accessible dynamic content and user interface components.
```
```suggestion
[Accessible Rich Internet Applications](https://www.w3.org/WAI/standards-guidelines/aria/), or WAI-ARIA is also a guideline that was created by WAI to provide an accessible experience for custom coded interactive experiences. In particular, WAI-ARIA gives guidance on how to create accessible dynamic content and user interface components.
```

This is a bit of a clarification, in that ARIA does not "increase" accessibility, it restores functionality to content that does not have an accessible experience.

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+[Accessible Rich Internet Applications](https://www.w3.org/WAI/standards-guidelines/aria/), or WAI-ARIA is also a guideline that was created by WAI to increase accessibility. In particular, WAI-ARIA gives guidance on how to create accessible dynamic content and user interface components.
+
+For example, ARIA can help developers divide the page into sections, kind of like a tab of index, so it’ll be easier for people to navigate the page. ARIA defines ways for functionality of the assistive technology. With the help of WAI-ARIA guidance and rules, developers can make websites accessible to all users.
```
I'm a little confused on what you mean by "tab of index". Could you clarify?

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+Some of the working group and interest groups listed on their web site are as follows:
+
+[Accessibility Guidelines Working Group (AG WG)](https://www.w3.org/WAI/GL/) (formerly the Web Content Accessibility Guidelines Working Group)
+
+[Accessible Platform Architectures (APA) Working Group](https://www.w3.org/WAI/APA/)
+
+[Accessible Rich Internet Applications (ARIA) Working Group](https://www.w3.org/WAI/ARIA/)
+
+[Education and Outreach Working Group (EOWG)](https://www.w3.org/WAI/about/groups/eowg/)
+
+[WAI Interest Group (WAI IG)](https://www.w3.org/WAI/about/groups/waiig/)
+
+[WAI websites](https://www.w3.org/WAI/about/participating/), explains how interested future members can join WAI to promote and develop accessible web. No matter your background, there is a group or interest group you can contribute.
```
```suggestion
- [Accessibility Guidelines Working Group (AG WG)](https://www.w3.org/WAI/GL/) (formerly the Web Content Accessibility Guidelines Working Group)
- [Accessible Platform Architectures (APA) Working Group](https://www.w3.org/WAI/APA/)
- [Accessible Rich Internet Applications (ARIA) Working Group](https://www.w3.org/WAI/ARIA/)
- [Education and Outreach Working Group (EOWG)](https://www.w3.org/WAI/about/groups/eowg/)
- [WAI Interest Group (WAI IG)](https://www.w3.org/WAI/about/groups/waiig/)
- [WAI websites](https://www.w3.org/WAI/about/participating/), explains how interested future members can join WAI to promote and develop accessible web. No matter your background, there is a group or interest group you can contribute.
```

Making this an unordered list, as it is a list of groups.

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+
+WAI touches most of the accessibility work that has been done. For example, designers and developers look for WCAG standards to see if their website is accessible or not. Developers also use WAI-ARIA’s guidance to know whether or not some interactions are accessible.
+
+Web Content Accessibility Guidelines, (WCAG) and Accessible Rich Internet Applications (WAI-Aria) Recommendations are some examples of WAI’s guidance on web accessibility.
```
```suggestion
Web Content Accessibility Guidelines, (WCAG) and Accessible Rich Internet Applications (WAI-ARIA) Recommendations are some examples of WAI’s guidance on web accessibility.
```

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,108 @@
+tags:
+  - concepts
+---
+
```
Can we have a very high-level sentence or two describing what we'll be covering in this post? You could even repeat what you wrote for the background metadata.

ericwbailey reviewed (changes requested):
Hey @ilknureren, this is looking great! I have some small styleguide stuff around capitalization and the like, and then a couple of questions/requests for updates for you. Looking forward to publishing this 🎉

ilknureren commented:
@ericwbailey Thank you for reviewing the article and making great suggestions!
I believe I addressed all your comments. 
Can I ask you to review the article one more time?

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,112 @@
+
+### Perceivable
+
+Users should be able to identify content and elements by means of senses. For example, if a website's navigation changes from page to page, users will have to relearn the navigation of the site, which will not be a great user experience.
```
```suggestion
People should be able to identify content and elements by means of their senses. For example, if a website's navigation changes from page to page, they will have to relearn the navigation of the site, which will not be a great experience.
```

Rephrasing to remove "user".

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,112 @@
+
+### Operarable:
+
+Users should successfully use controls, buttons, navigation, and other interactive elements on the page. One example of this would be that websites should be operable by a keyboard and the page should have the same functionalities it has when used by a mouse to navigate.
```
```suggestion
People should successfully use controls, buttons, navigation, and other interactive elements on the page. One example of this would be that websites should be operable by a keyboard and the page should have the same functionalities it has when a mouse is used to navigate.
```

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,112 @@
+
+### Understandable
+
+Users should be able to comprehend the content and learn from your website. One example of this would be if abbreviations or jargon is used, they should be defined for the user.
```
```suggestion
People should be able to comprehend the content and learn from your website. One example of this would defining abbreviations or jargon for the reader.
```

ericwbailey commented on src/posts/what-is-wai.md:
```diff
@@ -0,0 +1,112 @@
+
+### Robust:
+
+Websites should be able to be accessed by a wide variety of users and technologies. For example, websites should be accessible for screen readers and many different browser versions.
```
```suggestion
Websites should be able to be accessed by a wide variety of technologies. For example, websites should be accessible for screen readers, as well as many different browser versions.
```

ericwbailey reviewed (changes requested):
@ilknureren We're so close! A few last suggestions to use "person" terminology in place of "user".

The other thing, could you also add yourself to [`authors.json`](https://github.com/a11yproject/a11yproject.com/blob/main/src/_data/authors.json#L119)? Only `name` and `first` are required, but feel free to also add `website`, `photo`, and `biography` if that's something you feel comfortable doing.

This will make sure you get credit on the site for all your hard work 🎉

ilknureren commented:
> @ilknureren We're so close! A few last suggestions to use "person" terminology in place of "user".
> 
> The other thing, could you also add yourself to [`authors.json`](https://github.com/a11yproject/a11yproject.com/blob/main/src/_data/authors.json#L119)? Only `name` and `first` are required, but feel free to also add `website`, `photo`, and `biography` if that's something you feel comfortable doing.
> 
> This will make sure you get credit on the site for all your hard work 🎉

Woo Hoo! I added my information. Many thanks!

ericwbailey reviewed (approved):
@ilknureren  Amazing, thank you! A question for you: would you prefer this be published tomorrow, or next week?

Also, do you mind contacting us via [our contact form](https://www.a11yproject.com/contact/) so that we can coordinate payment?

ilknureren commented:
> @ilknureren Amazing, thank you! A question for you: would you prefer this be published tomorrow, or next week?
> 
> Also, do you mind contacting us via [our contact form](https://www.a11yproject.com/contact/) so that we can coordinate payment?

Thank you Eric! I am ok posting at any time!
